tests/__sf_section_elixir.test.fish

```diff
@@ -0,0 +1,109 @@
+source $DIRNAME/spacefish_test_setup.fish
+
+function setup
+	spacefish_test_setup
+	mock elixir -v 0 "echo \"Erlang/OTP 21 [erts-10.3.4] [source] [64-bit] [smp:8:8] [ds:8:8:10] [async-threads:1] [hipe] [dtrace]
```

Comment:
I recommend stubbing out the value of `$ELIXIR_VERSION` here or unsetting it entirely.